Unosquare.PiGpio.Workbench/Runners/LedBlinking.cs

```diff
@@ -13,22 +13,22 @@ public LedBlinking(bool isEnabled)
 
         protected override void OnSetup()
         {
-            _pin = Board.Pins[17];
+            _pin = Board.Pins[16];
         }
 
         protected override void DoBackgroundWork(CancellationToken ct)
         {
             while (!ct.IsCancellationRequested)
             {
                 _pin.Value = !_pin.Value;
-                Board.Timing.Sleep(500);
+                Thread.Sleep(500);
             }
         }
 
         protected override void Cleanup()
         {
-            _pin.Value = false;
-            Board.Timing.Sleep(200);
+            _pin.Value = !_pin.Value;
```

Comment:
This should be explicitly false, since you don't know the current status of the pin when the process ends